frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts

```diff
@@ -0,0 +1,76 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '@console/shared/src/test-views/dashboard-shared.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  mainHealtGreenSvg,
+  mainHealthCardStatus,
+  mainHealtRedSvg,
+  mainHealtYellowSvg,
+  noOutChange,
+  seeAllClose,
+  seeAllLink,
+  seeAllStorageGreenSvg,
+  seeAllStorageRedSvg,
+  seeAllStorageSection,
+  seeAllStorageYellowSvg,
+} from '../../views/ocp-dashboard-card-healthcheck.view';
+import { SECOND, OCP_TEXT_STATUS } from '../../utils/consts';
+
+describe('Check health data on main OCP dashboard and see all link.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/dashboards`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check main dashboard health is green', async () => {
+    expect(mainHealtGreenSvg.isPresent()).toBe(true);
+    expect(mainHealthCardStatus.getText()).toContain(OCP_TEXT_STATUS.HEALTHY);
+  });
+
+  it('Check main dashboard health floating see all card is green', async () => {
+    await click(seeAllLink);
+    await browser.wait(until.presenceOf(seeAllClose));
+    expect(seeAllStorageGreenSvg.isPresent()).toBe(true);
+    expect(seeAllStorageSection.getText()).toContain(OCP_TEXT_STATUS.HEALTHY);
+    await click(seeAllClose);
+  });
+
+  it('Check main dashboard health icon is yellow and cluster health is degraded', async () => {
+    await browser.wait(until.presenceOf(mainHealtGreenSvg));
+    noOutChange('set');
+    await browser.wait(until.not(until.presenceOf(mainHealtGreenSvg)));
+    await browser.sleep(2 * SECOND);
```

Comment:
I think we can wait on maiNHealthYellowSvg

Reply:
Didn't quite understand...It is waiting for it.

Reply:
Ok. Got you, I preferred waiting on green to disappear and then wait for couple of seconds for yellow. If we wait for green the expect after it is a double check and I've prefer using expect.